backends/qualcomm/runtime/backends/QnnBackendFactory.cpp

```diff
@@ -41,8 +39,8 @@ std::unique_ptr<BackendConfigParameters> QnnBackendFactory::Create(
       backend_params->backend_init_state_ = BackendInitializeState::INITIALIZED;
       return backend_params;
       break;
+    case kGpuBackend:
```

Comment:
Could you also remove `runtime/backends/QnnFactory.h`, since it's not necessary anymore.

Reply:
Okay!